pkg/specgen/generate/config_linux.go

```diff
@@ -170,6 +180,16 @@ func BlockAccessToKernelFilesystems(privileged, pidModeIsHost bool, g *generate.
 			g.AddLinuxReadonlyPaths(rp)
 		}
 	}
+
+	// mask the paths provided by the user
+	maskPaths := strings.Split(mask, ":")
+	for _, mp := range maskPaths {
+		if !path.IsAbs(mp) && mp != "" {
+			logrus.Errorf("path %q is not an absolute path, skipping...", mp)
```

Comment:
```suggestion
			logrus.Errorf("Path %q is not an absolute path, skipping...", mp)
```
Although we're not always consistent, we try to capitalize logs for easier reading.

Reply:
Fixed